src/index.js

```diff
@@ -0,0 +1,82 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+import path from 'path';
+import loaderUtils from 'loader-utils';
+
+export default function fileLoader(content) {
+  this.cacheable && this.cacheable(); // eslint-disable-line no-unused-expressions
```

Comment:
No, you're right about that. File loader was the guinea pig for defaults. It's being redone only to retain the git history on the lib files and to save me the trouble of having to rebase in changes on a file that was deleted on this branch :)